.github/workflows/test-prs.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: 'Dependency Review'
         uses: actions/dependency-review-action@v3
   test:
-    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@main
+    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@refactor-link-checker # TODO: change back to main
```

Comment:
Temporary change to test the test in this PR (but it doesn't work 😂 )

Reply:
you have to make some other changes to test, I'll ping you the runbook